src/components/Book.js

```diff
@@ -3,19 +3,51 @@ import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/apiReducer';
 
 function Book({ id, title, author }) {
+  const progress = Math.floor(Math.random() * 101);
+  const chapter = Math.floor(Math.random() * 10);
   const dispatch = useDispatch();
   const handleClick = () => {
     dispatch(deleteBook(id));
   };
   return (
     <div className="BOOK_CONTAINER">
-      <h2 className="Title">
-        {title}
-      </h2>
-      <h4 className="author">
-        {author}
-      </h4>
-      <button type="button" onClick={() => handleClick()} className="remove">Remove</button>
+      <div>
+        <span className="action">Action</span>
+        <h2 className="Title">
+          {title}
+        </h2>
+        <h4 className="author">
+          {author}
+        </h4>
+        <div>
+          {' '}
+          <span className="c-edit">Comments</span>
+          <button type="button" onClick={() => handleClick()} className="remove">Remove</button>
+          <span className="c-edit">Edit</span>
+        </div>
+      </div>
+      <div className="d-flex">
+        <img src="/circle.png" alt="progress" />
+        <div className="f-column">
+          {' '}
+          <span className="percentage">
+            {progress}
+            {' '}
+            %
+          </span>
+          <span className="completed">Completed</span>
+        </div>
+      </div>
+      <div className="f-column">
+        <span className="completed">Current Chapter</span>
+        <span>
+          Chapter
+          {' '}
+          { chapter }
+        </span>
+        <button type="button" className="ADD-BOOK update-progress">Update Progress</button>
+      </div>
+
```

Comment:
- [ ] Would you refactor your design to match the project requirement on this bottom form? The form of adding new book is still need to be worked on,

### Original design
![Screenshot from 2022-07-29 07-01-41](https://user-images.githubusercontent.com/56886064/181686213-8bfaa74e-495e-47e1-8378-517de99ec380.png)

### Your design
![Screenshot from 2022-07-29 06-57-20](https://user-images.githubusercontent.com/56886064/181686077-e03bab50-b43a-4d31-b6e7-43336bf06caf.png)